migrations/doorcontrol.sql

```diff
@@ -27,4 +27,17 @@ insert into log (id, name, action, result) values (0, 'tester', 'lock', 'succeed
 
 -- 2 down
 
-drop table if exists log;
\ No newline at end of file
+drop table if exists log;
+
+--3 up
+
+create table if not exists badges(
+    badge_id    int not null,
+    name        text not null,
+    authorized  int not null
+);
+
+insert into badges(badge_id, name, authorized) values (0123456, 'tester', 1);
+--3 down
```

Comment:
As this is all still the first Dev, all of these SQL statements can / should be combined into one version.  This is not strictly necessary, but if you maintain the art of this file it'll help to keep the size of it manageable.  Imagine this file 3 release later but with 18 db versions.  Not strictly a big deal, but more complicated...

I've thought about this a number of times, but haven't thought it thru...  It feels like it would be a good idea to keep the database version count in sync with program releases.  e.g.  As you commit to this repo for testing and sharing etc, it's still a WIP (work in progress).  There needn't be 6 different db versions when you haven't yet released this round of code.  Instead, if you combined all the 6 changes you make across the various Dev commits into a single db version, it'll be easier to mentally match db versions with program releases.